.gitmodules

```diff
@@ -1,3 +0,0 @@
-[submodule "json_schema"]
```

Comment:
So, if we remove the submodule for the OT json_schema, where do we get it from? what do we need to do to get future updates?

Reply:
Now we just fetch the required version `opentargets.json` from that repository. I added the command to wiki here: https://github.com/EBIvariation/eva-cttv-pipeline/wiki/How-to-submit-an-OpenTargets-batch#preparing-local-schema

Reply:
Could you please add to the documentation which folder exactly must the JSON file be downloaded to?

Reply:
It doesn't matter where you download it, because it is simply passed as a parameter at a later stage (as `--ot-schema opentargets.json`). I amended the documentation to clarify this and also reformatted the command string of `bin/evidence_string_generation.py` so that this is clearly visible.